adidas-ascii-logo/go/main_test.go

```diff
@@ -0,0 +1,18 @@
+package main
+import (
+  "fmt"
+  "testing"
+)
+
+func TestWidth2(t *testing.T) {
```

Comment:
Can we add a couple more tests for go?

Reply:
You can find all test cases in JS and TS folders.

Reply:
Done